folium/plugins/draw.py

```diff
@@ -43,37 +54,48 @@ class Draw(MacroElement):
                 });
                drawnItems.addLayer(layer);
              });
-
-        {{this._parent.get_name()}}.on('draw:created', function(e) {
-            drawnItems.addLayer(e.layer);
-        });
-
-        document.getElementById('export').onclick = function(e) {
-            var data = drawnItems.toGeoJSON();
-            var convertedData = 'text/json;charset=utf-8,' + encodeURIComponent(JSON.stringify(data));
-            document.getElementById('export').setAttribute('href', 'data:' + convertedData);
-            document.getElementById('export').setAttribute('download','data.geojson');
-        }
+            {{this._parent.get_name()}}.on('draw:created', function(e) {
+                drawnItems.addLayer(e.layer);
+            });
+            document.getElementById('export').onclick = function(e) {
+              var data = drawnItems.toGeoJSON();
+              var convertedData = 'text/json;charset=utf-8,' + encodeURIComponent(JSON.stringify(data));
+              document.getElementById('export').setAttribute('href', 'data:' + convertedData);
+              document.getElementById('export').setAttribute(
+                'download',
+                "{{kwargs.get('filename', 'data.geojson')}}"
+              );
+            }
             {% endmacro %}
             """)
 
-    def __init__(self, export=False):
+    js_url = 'https://cdnjs.cloudflare.com/ajax/libs/leaflet.draw/1.0.2/leaflet.draw.js'
+    css_url = 'https://cdnjs.cloudflare.com/ajax/libs/leaflet.draw/1.0.2/leaflet.draw.css'
+
+    def __init__(self, export=False, filename='data.geojson',
+                 position='topleft', draw=None, edit=None):
         super(Draw, self).__init__()
         self._name = 'DrawControl'
         self.export = export
+        self.filename = filename
+        self.position = position
+        self.draw = draw or {}
+        self.edit = edit or {}
 
     def render(self, **kwargs):
-        super(Draw, self).render()
+        super(Draw, self).render(
+            position=self.position,
+            draw=json.dumps(self.draw),
```

Comment:
These variables are already available in `self`, you don't have to pass them here. You can do the json dumping in `__init__`.